### ISSUE TITLE
Filters have a parent property

### ISSUE THREAD

**Issue by palfrey** [DavisRayM/django-filter-stubs#7]
It appears to be a `FilterSet` type, but not seeing on the definitions in here.

**Comment by DavisRayM** [comment]
I'll plan out sometime to update the package to ensure it's up to date with the latest django-filters sometime soon. Haven't been using the package of late.... PRs to update the stubs and maintain the repository would be greatly appreciated

**Comment by palfrey** [comment]
> I'll plan out sometime to update the package to ensure it's up to date with the latest django-filters sometime soon. Haven't been using the package of late.... PRs to update the stubs and maintain the repository would be greatly appreciated

Travis build doesn't appear to be working. Any interest in replacing those bits with github actions?

**Comment by DavisRayM** [comment]
Yes, I'll be happy to have the CI workflows moved to Github actions

**Comment by palfrey** [comment]
> Yes, I'll be happy to have the CI workflows moved to Github actions

Started work on this in https://github.com/DavisRayM/django-filter-stubs/pull/14 but running into a _lot_ of issues with Pipenv (see notes there). TBH, I'd lean towards replacing it. My current favourite option is [pip-tools](https://github.com/jazzband/pip-tools) but you may have other thoughts on more usable tools.

**Comment by DavisRayM** [comment]
I also currently prefer pip-tools I have no reservations with switch it out.
